COMETwebapp/Pages/Index.razor

```diff
@@ -22,30 +22,98 @@ Copyright (c) 2022 RHEA System S.A.
+                        <div class="col col-10">
+                            @context.User.Identity?.Name - @ISessionAnchor.Session.DataSourceUri
+                        </div>
+                        <div class="col">
+                            <a href="logout" class="m-left-10px">Log out</a>
+                        </div>
+                    </div>
                 </div>
-                <DxPopup HeaderText="Engineering Models" @bind-Visible="@PopupVisible">
-                    <OpenModel />
-                </DxPopup>
+                <DxTreeView CssClass="m-top-10px">
+                    <Nodes>
+                        <DxTreeViewNode Name="ModelView" Text="Open a Model" Expanded CssClass="color-title font-weight-bold">
+                            <Nodes>
+                                <DxTreeViewNode CssClass="font-weight-100">
+                                    <Template Context="second_context">
+                                        <OpenModel />
+                                    </Template>
+                                </DxTreeViewNode>
+                            </Nodes>
+                        </DxTreeViewNode>
+                    </Nodes>
+                </DxTreeView>
             </Authorized>
             <NotAuthorized>
-                <div id="unauthorized-notice">Welcome to the COMET Web Application!</div>                
+                <div class="text-align-center font-weight-bold">Connect and Open a model.</div>
+                <DxTreeView CssClass="m-top-10px">
+                    <Nodes>
+                        <DxTreeViewNode Name="ConnectView" Text="Connect" Expanded CssClass="color-title font-weight-bold">
+                            <Nodes>
+                                <DxTreeViewNode CssClass="font-weight-100">
+                                    <Template Context="second_context">
+                                        <Login />
+                                    </Template>
+                                </DxTreeViewNode>
+                            </Nodes>
+                        </DxTreeViewNode>
+                    </Nodes>
+                </DxTreeView>              
             </NotAuthorized>
-        </AuthorizeView>    
-    </h1>
+        </AuthorizeView>
+        </div>
+    </div>
 </div>
+    
 
 @code{
     /// <summary>
     /// Boolean to determine if the Popup OpenModel is visible
     /// </summary>
-    bool PopupVisible = false;
+    bool modelPopupVisible = false;
+
+    /// <summary>
+	/// The Version of the COMET-WEB CE application
+	/// </summary>
+	private string cometWebVersion = string.Empty;
+	
+	/// <summary>
+	/// The license text as retrieved from the GitHub repository
+	/// </summary>
+	private string license = string.Empty;
+
+	/// <summary>
+	/// Initializes the component
+	/// </summary>
+	protected override async Task OnInitializedAsync()
+	{
+		this.cometWebVersion = typeof(Program).Assembly.GetName().Version.ToString();
```

Comment:
please abstract this into an injectable service that is called, something like `IVersionService.GetVersion()`